gpustack/schemas/models.py

```diff
@@ -57,6 +57,7 @@ class ModelBase(SQLModel, ModelSource):
     worker_selector: Optional[Dict[str, str]] = Field(
         sa_column=Column(JSON), default={}
     )
+    gpu_selector: Optional[str] = None
```

Comment:
How to extend if we need multi gpus? Can we use worker_name and gpu_index to pick a gpu?

Reply:
Prefer change to gpu_selector to a List[str] while we need multi GPUs, since the GPU index is consistent with the info user gets from tools like nvidia-smi, so while we support a machine with different GPUs, it‘s easier to tell them apart by id.